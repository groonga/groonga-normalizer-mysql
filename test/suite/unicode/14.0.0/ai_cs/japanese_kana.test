register normalizers/mysql

normalize \
  'NormalizerMySQLUnicode("version", "14.0.0", "case_sensitive", true)' \
  "はハ"
