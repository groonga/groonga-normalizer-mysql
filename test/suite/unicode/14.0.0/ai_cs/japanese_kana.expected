register normalizers/mysql
[[0,0.0,0.0],true]
normalize   'NormalizerMySQLUnicode("version", "14.0.0", "case_sensitive", true)'   "はハ"
[[0,0.0,0.0],{"normalized":"はハ","types":[],"checks":[]}]
